source/Engine.cpp

```diff
@@ -286,7 +284,7 @@ void Engine::Place()
 		planetRadius = object->Radius();
 	}
 	
-	// Give each non-carried, special ship we just added a random heading and position.
+	// Give each non-carried player's ship a random heading and position.
```

Comment:
This function also seems to place non-player NPC ships.

Reply:
Look at lines 263-267, there the ship list is filled only with player's ships.

Reply:
At least I hope the list is not modified elsewhere.

Reply:
Yep, you are correct, it does add NPCs.